external-crates/move/README.md

```diff
@@ -4,21 +4,24 @@ title: Move Language
 custom_edit_url: https://github.com/move-language/move/edit/main/language/README.md
 ---
 
-
-Move is a new programming language developed to provide a safe and programmable foundation for the Diem Blockchain.
+Move is a new programming language developed to provide a safe and programmable foundation for smart contract development.
 
 ## Overview
 
-The Move language directory consists of four main parts:
-
-- [virtual machine](move-vm/) (VM) &mdash; contains the bytecode format, a bytecode interpreter, and infrastructure for executing a block of transactions. This directory also contains the infrastructure to generate the genesis block.
-
-- [bytecode verifier](move-bytecode-verifier/) &mdash; contains a static analysis tool for rejecting invalid Move bytecode. The virtual machine runs the bytecode verifier on any new Move code it encounters before executing it. The compiler runs the bytecode verifier on its output and surfaces the errors to the programmer.
-
-- [move-compiler](move-compiler/) &mdash; contains the Move source language compiler.
+The Move language directory consists of three distinct parts: the Move bytecode language, the Move IR, and the Move source language.
```

Comment:
```suggestion
The Move language directory consists of three distinct parts: the Move bytecode language, the Move intermediate representation (IR), and the Move source language.
```
Suggest this on first use for those folks that might not immediately know what IR stands for and have to use Google or AI to be sure. I'm not one of those people, of course, but there might be some out there...